includes/Modules/Analytics.php

```diff
@@ -117,8 +117,6 @@ public function register() {
 		add_action( 'wp_head', $this->get_method_proxy( 'print_tracking_opt_out' ), 0 );
 		// For Web Stories plugin.
 		add_action( 'web_stories_story_head', $this->get_method_proxy( 'print_tracking_opt_out' ), 0 );
-		// Analytics tag placement logic.
-		add_action( 'template_redirect', $this->get_method_proxy( 'register_tag' ) );
```

Comment:
Just noting that this is an intentional change which removes the tagging for UA tag classes making the decoupled GA4 equivalents standalone tagging for GA going forward.